manim/scene/scene.py

```diff
@@ -873,6 +880,16 @@ def play_internal(self, skip_rendering=False):
             self.update_mobjects(0)
         self.renderer.static_image = None
 
+    def interact(self):
```

Comment:
Maybe add `interact` to the local namespace of Scene.embed (i.e. the loop on line 918)?

Reply:
I'm not sure what you mean exactly, but I also don't think it will allow interactivity during `embed()`.

Reply:
You can already call `self.interact()` while in the console, and since `play`, `wait` etc. can be called without the `self.` already, I thought it might be better for consistency.

Reply:
I see, that should be easy